File: MiniDOMOffice/Magazzino/Inventario/OggettoInventariatoCursor.vb
Imports minidom.Databases
Imports minidom.Sistema
Imports minidom.Anagrafica



Partial Public Class Office
 
    Public Class OggettoInventariatoCursor
        Inherits DBObjectCursorPO(Of OggettoInventariato)

        Private m_IDArticolo As New CCursorField(Of Integer)("IDArticolo")
        Private m_Codice As New CCursorFieldObj(Of String)("Codice")
        Private m_Nome As New CCursorFieldObj(Of String)("Nome")
        Private m_NomeArticolo As New CCursorFieldObj(Of String)("NomeArticolo")
        Private m_Categoria As New CCursorFieldObj(Of String)("Categoria")
        Private m_Marca As New CCursorFieldObj(Of String)("Marca")
        Private m_Modello As New CCursorFieldObj(Of String)("Modello")
        Private m_Seriale As New CCursorFieldObj(Of String)("Seriale")
        Private m_Descrizione As New CCursorFieldObj(Of String)("Descrizione")
        Private m_IconURL As New CCursorFieldObj(Of String)("IconURL")
        Private m_StatoAttuale As New CCursorField(Of StatoOggettoInventariato)("StatoAttuale")
        Private m_DettaglioStatoAttuale As New CCursorFieldObj(Of String)("DettaglioStatoAttuale")
        Private m_ValoreStimato As New CCursorField(Of Decimal)("ValoreStimato")
        Private m_DataValutazione As New CCursorField(Of Date)("DataValutazione")
        Private m_TipoInUsoDa As New CCursorFieldObj(Of String)("TipoInUsoDa")
        Private m_IDInUsoDa As New CCursorField(Of Integer)("InUsoDaID")
        Private m_NomeInUsoDa As New CCursorFieldObj(Of String)("NomeInUsoDa")

        Private m_DataProduzione As New CCursorField(Of Date)("DataProduzione")
        Private m_DataAcquisto As New CCursorField(Of Date)("DataAscquisto")
        Private m_TipoDocumentoAcquisto As New CCursorFieldObj(Of String)("TipoDocumentoAcquisto")
        Private m_NumeroDocumentoAcquisto As New CCursorFieldObj(Of String)("NumeroDocumentoAcquisto")
        Private m_StatoAcquisto As New CCursorField(Of StatoAcquistoOggettoInventariato)("StatoAcquisto")
        Private m_DettaglioStatoAcquisto As New CCursorFieldObj(Of String)("DettaglioStatoAcquisto")
        Private m_AcquistatoDaID As New CCursorField(Of Integer)("AcquistatoDaID")
        Private m_NomeAcquistatoDa As New CCursorFieldObj(Of String)("NomeAcquistatoDa")
        Private m_PrezzoAcquisto As New CCursorField(Of Decimal)("PrezzoAcquisto")
        Private m_AliquotaIVA As New CCursorField(Of Single)("AliquotaIVA")

        Private m_IDUfficioOriginale As New CCursorField(Of Integer)("IDUfficioOriginale")
        Private m_NomeUfficioOriginale As New CCursorFieldObj(Of String)("NomeUfficioOriginale")

        Private m_CodiceScaffale As New CCursorFieldObj(Of String)("CodiceScaffale")
        Private m_CodiceReparto As New CCursorFieldObj(Of String)("CodiceReparto")

        Private m_DataDismissione As New CCursorField(Of Date)("DataDismissione")
        Private m_DismessoDaID As New CCursorField(Of Integer)("DismessoDaID")
        Private m_NomeDismessoDa As New CCursorFieldObj(Of String)("NomeDismessoDa")
        Private m_MotivoDismissione As New CCursorFieldObj(Of String)("MotivoDismissione")
        Private m_DettaglioDismissione As New CCursorFieldObj(Of String)("DettaglioDismissione")
        Private m_ValoreDismissione As New CCursorField(Of Decimal)("ValoreDismissione")
        Private m_AliquotaIVADismissione As New CCursorField(Of Single)("AliquotaIVADismissione")
        Private m_Flags As New CCursorField(Of FlagsOggettoInventariato)("Flags")
        Private m_IDOrdineAcquisto As New CCursorField(Of Integer)("IDOrdineAcquisto")
        Private m_IDDocumentoAcquisto As New CCursorField(Of Integer)("IDDocumentoAcquisto")
        Private m_IDSpedizione As New CCursorField(Of Integer)("IDSpedizione")
        Private m_CodiceRFID As New CCursorFieldObj(Of String)("CodiceRFID")

        Public Sub New()
        End Sub

        Public ReadOnly Property CodiceRFID As CCursorFieldObj(Of String)
            Get
                Return Me.m_CodiceRFID
            End Get
        End Property

        Public ReadOnly Property IDOrdineAcquisto As CCursorField(Of Integer)
            Get
                Return Me.m_IDOrdineAcquisto
            End Get
        End Property

        Public ReadOnly Property IDDocumentoAcquisto As CCursorField(Of Integer)
            Get
                Return Me.m_IDDocumentoAcquisto
            End Get
        End Property

        Public ReadOnly Property IDSpedizione As CCursorField(Of Integer)
            Get
                Return Me.m_IDSpedizione
            End Get
        End Property

        Public ReadOnly Property IDArticolo As CCursorField(Of Integer)
            Get
                Return Me.m_IDArticolo
            End Get
        End Property

        Public ReadOnly Property Codice As CCursorFieldObj(Of String)
            Get
                Return Me.m_Codice
            End Get
        End Property

        Public ReadOnly Property Nome As CCursorFieldObj(Of String)
            Get
                Return Me.m_Nome
            End Get
        End Property

        Public ReadOnly Property NomeArticolo As CCursorFieldObj(Of String)
            Get
                Return Me.m_NomeArticolo
            End Get
        End Property

        Public ReadOnly Property Categoria As CCursorFieldObj(Of String)
            Get
                Return Me.m_Categoria
            End Get
        End Property

        Public ReadOnly Property Marca As CCursorFieldObj(Of String)
            Get
                Return Me.m_Marca
            End Get
        End Property

        Public ReadOnly Property Modello As CCursorFieldObj(Of String)
            Get
                Return Me.m_Modello
            End Get
        End Property

        Public ReadOnly Property Seriale As CCursorFieldObj(Of String)
            Get
                Return Me.m_Seriale
            End Get
        End Property

        Public ReadOnly Property Descrizione As CCursorFieldObj(Of String)
            Get
                Return Me.m_Descrizione
            End Get
        End Property

        Public ReadOnly Property IconURL As CCursorFieldObj(Of String)
            Get
                Return Me.m_IconURL
            End Get
        End Property

        Public ReadOnly Property StatoAttuale As CCursorField(Of StatoOggettoInventariato)
            Get
                Return Me.m_StatoAttuale
            End Get
        End Property

        Public ReadOnly Property DettaglioStatoAttuale As CCursorFieldObj(Of String)
            Get
                Return Me.m_DettaglioStatoAttuale
            End Get
        End Property

        Public ReadOnly Property ValoreStimato As CCursorField(Of Decimal)
            Get
                Return Me.m_ValoreStimato
            End Get
        End Property

        Public ReadOnly Property DataValutazione As CCursorField(Of Date)
            Get
                Return Me.m_DataValutazione
            End Get
        End Property

        Public ReadOnly Property TipoInUsoDa As CCursorFieldObj(Of String)
            Get
                Return Me.m_TipoInUsoDa
            End Get
        End Property

        Public ReadOnly Property IDInUsoDa As CCursorField(Of Integer)
            Get
                Return Me.m_IDInUsoDa
            End Get
        End Property

        Public ReadOnly Property NomeInUsoDa As CCursorFieldObj(Of String)
            Get
                Return Me.m_NomeInUsoDa
            End Get
        End Property

        Public ReadOnly Property DataProduzione As CCursorField(Of Date)
            Get
                Return Me.m_DataProduzione
            End Get
        End Property

        Public ReadOnly Property DataAcquisto As CCursorField(Of Date)
            Get
                Return Me.m_DataAcquisto
            End Get
        End Property

        Public ReadOnly Property TipoDocumentoAcquisto As CCursorFieldObj(Of String)
            Get
                Return Me.m_TipoDocumentoAcquisto
            End Get
        End Property

        Public ReadOnly Property NumeroDocumentoAcquisto As CCursorFieldObj(Of String)
            Get
                Return Me.m_NumeroDocumentoAcquisto
            End Get
        End Property

        Public ReadOnly Property StatoAcquisto As CCursorField(Of StatoAcquistoOggettoInventariato)
            Get
                Return Me.m_StatoAcquisto
            End Get
        End Property

        Public ReadOnly Property DettaglioStatoAcquisto As CCursorFieldObj(Of String)
            Get
                Return Me.m_DettaglioStatoAcquisto
            End Get
        End Property

        Public ReadOnly Property AcquistatoDaID As CCursorField(Of Integer)
            Get
                Return Me.m_AcquistatoDaID
            End Get
        End Property

        Public ReadOnly Property NomeAcquistatoDa As CCursorFieldObj(Of String)
            Get
                Return Me.m_NomeAcquistatoDa
            End Get
        End Property

        Public ReadOnly Property PrezzoAcquisto As CCursorField(Of Decimal)
            Get
                Return Me.m_PrezzoAcquisto
            End Get
        End Property

        Public ReadOnly Property AliquotaIVA As CCursorField(Of Single)
            Get
                Return Me.m_AliquotaIVA
            End Get
        End Property

        Public ReadOnly Property IDUfficioOriginale As CCursorField(Of Integer)
            Get
                Return Me.m_IDUfficioOriginale
            End Get
        End Property

        Public ReadOnly Property NomeUfficioOriginale As CCursorFieldObj(Of String)
            Get
                Return Me.m_NomeUfficioOriginale
            End Get
        End Property

        Public ReadOnly Property CodiceScaffale As CCursorFieldObj(Of String)
            Get
                Return Me.m_CodiceScaffale
            End Get
        End Property

        Public ReadOnly Property CodiceReparto As CCursorFieldObj(Of String)
            Get
                Return Me.m_CodiceReparto
            End Get
        End Property

        Public ReadOnly Property DataDismissione As CCursorField(Of Date)
            Get
                Return Me.m_DataDismissione
            End Get
        End Property

        Public ReadOnly Property DismessoDaID As CCursorField(Of Integer)
            Get
                Return Me.m_DismessoDaID
            End Get
        End Property

        Public ReadOnly Property NomeDismessoDa As CCursorFieldObj(Of String)
            Get
                Return Me.m_NomeDismessoDa
            End Get
        End Property

        Public ReadOnly Property MotivoDismissione As CCursorFieldObj(Of String)
            Get
                Return Me.m_MotivoDismissione
            End Get
        End Property

        Public ReadOnly Property DettaglioDismissione As CCursorFieldObj(Of String)
            Get
                Return Me.m_DettaglioDismissione
            End Get
        End Property

        Public ReadOnly Property ValoreDismissione As CCursorField(Of Decimal)
            Get
                Return Me.m_ValoreDismissione
            End Get
        End Property

        Public ReadOnly Property AliquotaIVADismissione As CCursorField(Of Single)
            Get
                Return Me.m_AliquotaIVADismissione
            End Get
        End Property

        Public ReadOnly Property Flags As CCursorField(Of FlagsOggettoInventariato)
            Get
                Return Me.m_Flags
            End Get
        End Property

        Protected Overrides Function GetConnection() As CDBConnection
            Return Office.Database
        End Function

        Protected Overrides Function GetModule() As CModule
            Return Office.OggettiInventariati.Module
        End Function

        Public Overrides Function GetTableName() As String
            Return "tbl_OfficeOggettiInventariati"
        End Function


    End Class

End Class

File: MiniDOMOffice/Magazzino/Inventario/OggettoInventariatoCursor.cs
using System;
using System.Collections;
using System.Collections.Generic;
using DMD;
using DMD.XML;
using DMD.Databases;
using DMD.Databases.Collections;
using minidom;
using minidom.repositories;
using static minidom.Sistema;
using static minidom.Anagrafica;
using static minidom.Store;

namespace minidom
{
    public partial class Store
    {

        /// <summary>
        /// Cursore di <see cref="OggettoInventariato"/>
        /// </summary>

        [Serializable]
        public class OggettoInventariatoCursor
            : minidom.Databases.DBObjectCursorPO<OggettoInventariato>
        {
            private DBCursorField<int> m_IDArticolo = new DBCursorField<int>("IDArticolo");
            private DBCursorStringField m_Codice = new DBCursorStringField("Codice");
            private DBCursorStringField m_Nome = new DBCursorStringField("Nome");
            private DBCursorStringField m_NomeArticolo = new DBCursorStringField("NomeArticolo");
            private DBCursorStringField m_Categoria = new DBCursorStringField("Categoria");
            private DBCursorStringField m_Marca = new DBCursorStringField("Marca");
            private DBCursorStringField m_Modello = new DBCursorStringField("Modello");
            private DBCursorStringField m_Seriale = new DBCursorStringField("Seriale");
            private DBCursorStringField m_Descrizione = new DBCursorStringField("Descrizione");
            private DBCursorStringField m_IconURL = new DBCursorStringField("IconURL");
            private DBCursorField<StatoOggettoInventariato> m_StatoAttuale = new DBCursorField<StatoOggettoInventariato>("StatoAttuale");
            private DBCursorStringField m_DettaglioStatoAttuale = new DBCursorStringField("DettaglioStatoAttuale");
            private DBCursorField<decimal> m_ValoreStimato = new DBCursorField<decimal>("ValoreStimato");
            private DBCursorField<DateTime> m_DataValutazione = new DBCursorField<DateTime>("DataValutazione");
            private DBCursorStringField m_TipoInUsoDa = new DBCursorStringField("TipoInUsoDa");
            private DBCursorField<int> m_IDInUsoDa = new DBCursorField<int>("InUsoDaID");
            private DBCursorStringField m_NomeInUsoDa = new DBCursorStringField("NomeInUsoDa");
            private DBCursorField<DateTime> m_DataProduzione = new DBCursorField<DateTime>("DataProduzione");
            private DBCursorField<DateTime> m_DataAcquisto = new DBCursorField<DateTime>("DataAscquisto");
            private DBCursorStringField m_TipoDocumentoAcquisto = new DBCursorStringField("TipoDocumentoAcquisto");
            private DBCursorStringField m_NumeroDocumentoAcquisto = new DBCursorStringField("NumeroDocumentoAcquisto");
            private DBCursorField<StatoAcquistoOggettoInventariato> m_StatoAcquisto = new DBCursorField<StatoAcquistoOggettoInventariato>("StatoAcquisto");
            private DBCursorStringField m_DettaglioStatoAcquisto = new DBCursorStringField("DettaglioStatoAcquisto");
            private DBCursorField<int> m_AcquistatoDaID = new DBCursorField<int>("AcquistatoDaID");
            private DBCursorStringField m_NomeAcquistatoDa = new DBCursorStringField("NomeAcquistatoDa");
            private DBCursorField<decimal> m_PrezzoAcquisto = new DBCursorField<decimal>("PrezzoAcquisto");
            private DBCursorField<double> m_AliquotaIVA = new DBCursorField<double>("AliquotaIVA");
            private DBCursorField<int> m_IDUfficioOriginale = new DBCursorField<int>("IDUfficioOriginale");
            private DBCursorStringField m_NomeUfficioOriginale = new DBCursorStringField("NomeUfficioOriginale");
            private DBCursorStringField m_CodiceScaffale = new DBCursorStringField("CodiceScaffale");
            private DBCursorStringField m_CodiceReparto = new DBCursorStringField("CodiceReparto");
            private DBCursorField<DateTime> m_DataDismissione = new DBCursorField<DateTime>("DataDismissione");
            private DBCursorField<int> m_DismessoDaID = new DBCursorField<int>("DismessoDaID");
            private DBCursorStringField m_NomeDismessoDa = new DBCursorStringField("NomeDismessoDa");
            private DBCursorStringField m_MotivoDismissione = new DBCursorStringField("MotivoDismissione");
            private DBCursorStringField m_DettaglioDismissione = new DBCursorStringField("DettaglioDismissione");
            private DBCursorField<decimal> m_ValoreDismissione = new DBCursorField<decimal>("ValoreDismissione");
            private DBCursorField<double> m_AliquotaIVADismissione = new DBCursorField<double>("AliquotaIVADismissione");
            private DBCursorField<FlagsOggettoInventariato> m_Flags = new DBCursorField<FlagsOggettoInventariato>("Flags");
            private DBCursorField<int> m_IDOrdineAcquisto = new DBCursorField<int>("IDOrdineAcquisto");
            private DBCursorField<int> m_IDDocumentoAcquisto = new DBCursorField<int>("IDDocumentoAcquisto");
            private DBCursorField<int> m_IDSpedizione = new DBCursorField<int>("IDSpedizione");
            private DBCursorStringField m_CodiceRFID = new DBCursorStringField("CodiceRFID");

            /// <summary>
            /// Costruttore
            /// </summary>
            public OggettoInventariatoCursor()
            {
            }

            /// <summary>
            /// CodiceRFID
            /// </summary>
            public DBCursorStringField CodiceRFID
            {
                get
                {
                    return m_CodiceRFID;
                }
            }

            /// <summary>
            /// IDOrdineAcquisto
            /// </summary>
            public DBCursorField<int> IDOrdineAcquisto
            {
                get
                {
                    return m_IDOrdineAcquisto;
                }
            }

            /// <summary>
            /// IDDocumentoAcquisto
            /// </summary>
            public DBCursorField<int> IDDocumentoAcquisto
            {
                get
                {
                    return m_IDDocumentoAcquisto;
                }
            }

            /// <summary>
            /// IDSpedizione
            /// </summary>
            public DBCursorField<int> IDSpedizione
            {
                get
                {
                    return m_IDSpedizione;
                }
            }

            /// <summary>
            /// IDArticolo
            /// </summary>
            public DBCursorField<int> IDArticolo
            {
                get
                {
                    return m_IDArticolo;
                }
            }

            /// <summary>
            /// Codice
            /// </summary>
            public DBCursorStringField Codice
            {
                get
                {
                    return m_Codice;
                }
            }

            /// <summary>
            /// Nome
            /// </summary>
            public DBCursorStringField Nome
            {
                get
                {
                    return m_Nome;
                }
            }

            /// <summary>
            /// NomeArticolo
            /// </summary>
            public DBCursorStringField NomeArticolo
            {
                get
                {
                    return m_NomeArticolo;
                }
            }

            /// <summary>
            /// Categoria
            /// </summary>
            public DBCursorStringField Categoria
            {
                get
                {
                    return m_Categoria;
                }
            }

            /// <summary>
            /// Marca
            /// </summary>
            public DBCursorStringField Marca
            {
                get
                {
                    return m_Marca;
                }
            }

            /// <summary>
            /// Modello
            /// </summary>
            public DBCursorStringField Modello
            {
                get
                {
                    return m_Modello;
                }
            }

            /// <summary>
            /// Seriale
            /// </summary>
            public DBCursorStringField Seriale
            {
                get
                {
                    return m_Seriale;
                }
            }

            /// <summary>
            /// Descrizione
            /// </summary>
            public DBCursorStringField Descrizione
            {
                get
                {
                    return m_Descrizione;
                }
            }

            /// <summary>
            /// IconURL
            /// </summary>
            public DBCursorStringField IconURL
            {
                get
                {
                    return m_IconURL;
                }
            }

            /// <summary>
            /// StatoAttuale
            /// </summary>
            public DBCursorField<StatoOggettoInventariato> StatoAttuale
            {
                get
                {
                    return m_StatoAttuale;
                }
            }

            /// <summary>
            /// DettaglioStatoAttuale
            /// </summary>
            public DBCursorStringField DettaglioStatoAttuale
            {
                get
                {
                    return m_DettaglioStatoAttuale;
                }
            }

            /// <summary>
            /// ValoreStimato
            /// </summary>
            public DBCursorField<decimal> ValoreStimato
            {
                get
                {
                    return m_ValoreStimato;
                }
            }

            /// <summary>
            /// DataValutazione
            /// </summary>
            public DBCursorField<DateTime> DataValutazione
            {
                get
                {
                    return m_DataValutazione;
                }
            }

            /// <summary>
            /// TipoInUsoDa
            /// </summary>
            public DBCursorStringField TipoInUsoDa
            {
                get
                {
                    return m_TipoInUsoDa;
                }
            }

            /// <summary>
            /// IDInUsoDa
            /// </summary>
            public DBCursorField<int> IDInUsoDa
            {
                get
                {
                    return m_IDInUsoDa;
                }
            }

            /// <summary>
            /// NomeInUsoDa
            /// </summary>
            public DBCursorStringField NomeInUsoDa
            {
                get
                {
                    return m_NomeInUsoDa;
                }
            }

            /// <summary>
            /// DataProduzione
            /// </summary>
            public DBCursorField<DateTime> DataProduzione
            {
                get
                {
                    return m_DataProduzione;
                }
            }

            /// <summary>
            /// DataAcquisto
            /// </summary>
            public DBCursorField<DateTime> DataAcquisto
            {
                get
                {
                    return m_DataAcquisto;
                }
            }

            /// <summary>
            /// TipoDocumentoAcquisto
            /// </summary>
            public DBCursorStringField TipoDocumentoAcquisto
            {
                get
                {
                    return m_TipoDocumentoAcquisto;
                }
            }

            /// <summary>
            /// NumeroDocumentoAcquisto
            /// </summary>
            public DBCursorStringField NumeroDocumentoAcquisto
            {
                get
                {
                    return m_NumeroDocumentoAcquisto;
                }
            }

            /// <summary>
            /// StatoAcquisto
            /// </summary>
            public DBCursorField<StatoAcquistoOggettoInventariato> StatoAcquisto
            {
                get
                {
                    return m_StatoAcquisto;
                }
            }

            /// <summary>
            /// DettaglioStatoAcquisto
            /// </summary>
            public DBCursorStringField DettaglioStatoAcquisto
            {
                get
                {
                    return m_DettaglioStatoAcquisto;
                }
            }

            /// <summary>
            /// AcquistatoDaID
            /// </summary>
            public DBCursorField<int> AcquistatoDaID
            {
                get
                {
                    return m_AcquistatoDaID;
                }
            }

            /// <summary>
            /// NomeAcquistatoDa
            /// </summary>
            public DBCursorStringField NomeAcquistatoDa
            {
                get
                {
                    return m_NomeAcquistatoDa;
                }
            }

            /// <summary>
            /// PrezzoAcquisto
            /// </summary>
            public DBCursorField<decimal> PrezzoAcquisto
            {
                get
                {
                    return m_PrezzoAcquisto;
                }
            }

            /// <summary>
            /// AliquotaIVA
            /// </summary>
            public DBCursorField<double> AliquotaIVA
            {
                get
                {
                    return m_AliquotaIVA;
                }
            }

            /// <summary>
            /// IDUfficioOriginale
            /// </summary>
            public DBCursorField<int> IDUfficioOriginale
            {
                get
                {
                    return m_IDUfficioOriginale;
                }
            }

            /// <summary>
            /// NomeUfficioOriginale
            /// </summary>
            public DBCursorStringField NomeUfficioOriginale
            {
                get
                {
                    return m_NomeUfficioOriginale;
                }
            }

            /// <summary>
            /// CodiceScaffale
            /// </summary>
            public DBCursorStringField CodiceScaffale
            {
                get
                {
                    return m_CodiceScaffale;
                }
            }

            /// <summary>
            /// CodiceReparto
            /// </summary>
            public DBCursorStringField CodiceReparto
            {
                get
                {
                    return m_CodiceReparto;
                }
            }

            /// <summary>
            /// DataDismissione
            /// </summary>
            public DBCursorField<DateTime> DataDismissione
            {
                get
                {
                    return m_DataDismissione;
                }
            }

            /// <summary>
            /// DismessoDaID
            /// </summary>
            public DBCursorField<int> DismessoDaID
            {
                get
                {
                    return m_DismessoDaID;
                }
            }

            /// <summary>
            /// NomeDismessoDa
            /// </summary>
            public DBCursorStringField NomeDismessoDa
            {
                get
                {
                    return m_NomeDismessoDa;
                }
            }

            /// <summary>
            /// MotivoDismissione
            /// </summary>
            public DBCursorStringField MotivoDismissione
            {
                get
                {
                    return m_MotivoDismissione;
                }
            }

            /// <summary>
            /// DettaglioDismissione
            /// </summary>
            public DBCursorStringField DettaglioDismissione
            {
                get
                {
                    return m_DettaglioDismissione;
                }
            }

            /// <summary>
            /// ValoreDismissione
            /// </summary>
            public DBCursorField<decimal> ValoreDismissione
            {
                get
                {
                    return m_ValoreDismissione;
                }
            }

            /// <summary>
            /// AliquotaIVADismissione
            /// </summary>
            public DBCursorField<double> AliquotaIVADismissione
            {
                get
                {
                    return m_AliquotaIVADismissione;
                }
            }

            /// <summary>
            /// Flags
            /// </summary>
            public DBCursorField<FlagsOggettoInventariato> Flags
            {
                get
                {
                    return m_Flags;
                }
            }

            /// <summary>
            /// Repository
            /// </summary>
            /// <returns></returns>
            protected override CModulesClass GetModule()
            {
                return minidom.Store.OggettiInventariati;
            }

            //public override string GetTableName()
            //{
            //    return "tbl_OfficeOggettiInventariati";
            //}
        }
    }
}